examples/example_artist_links.ipynb

```diff
@@ -0,0 +1,93 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "27246622",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "from pathlib import Path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "75f62754",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.append('../artscraper/')"
```

Comment:
This is not necessary, just install the artscraper package with `pip install .` or `pip install -e .

Reply:
I tried to integrate my code into the package by changing __init.py__, but I don't seem to have done it quite right. Pylint is now giving a cyclic import error:

************* Module artscraper.wikiart
artscraper/wikiart.py:1:0: R0401: Cyclic import (artscraper -> artscraper.artist_links) (cyclic-import)